src/components/Header/WhatNew.tsx

```
@@ -143,12 +132,11 @@
             date="2023-11-20"
             li={[
               <>
-                add article:{' '}
-                <DialogClose asChild>
-                  <Link href="/articles/Trying-less-energy-coding">
-                    Trying less energy coding
-                  </Link>
-                </DialogClose>
+                add article:
+                <Link href="https://laststance.io/articles/Trying-less-energy-coding">
+                  Trying less energy coding
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)